.github/workflows/ci.yml

```
@@ -18,7 +18,7 @@ env:
   RUSTFLAGS: "-D warnings"
   FUEL_CORE_VERSION: 0.10.1
   RUST_VERSION: 1.61.0
-  FORC_VERSION: 0.20.2
+  FORC_VERSION: 0.22.0
```

```suggestion
  FORC_VERSION: 0.22.1
```
Not too important but we've had to add a last minute change.